Makefile

```diff
@@ -75,17 +73,15 @@ clean :
 	-kubectl delete -f deploy/webv --ignore-not-found=true
 	-kubectl delete -f deploy/ngsa-memory --ignore-not-found=true
 	-kubectl delete ns monitoring --ignore-not-found=true
-	-kubectl delete -f deploy/fluentbit/fluentbit-pod.yaml --ignore-not-found=true
-	-kubectl delete secret log-secrets --ignore-not-found=true
+	-kubectl delete ns logging --ignore-not-found=true
 
 	# show running pods
 	@kubectl get po -A
 
 app :
-	# build the local image and load into kind
-	docker build ../ngsa-app -t ngsa-app:local
-
-	kind load docker-image ngsa-app:local
+	# build push the local image
+	docker build ../ngsa-app -t k3d-registry.localhost:5000/ngsa-app:local
+	docker push k3d-registry.localhost:5000/ngsa-app:local
 
 	# delete WebV
 	-kubectl delete -f deploy/webv --ignore-not-found=true
```

Comment:
The README says to run `make webv` and then `make app` right after, but there isn't a `-kubectl delete -f deploy/webv-local --ignore-not-found=true` in `make app`

Reply:
deleting deploy/webv will is the same as deleting deploy/webv-local - I'll add a comment that it will work for either for clarity